### PR TITLE
added llama3 variants

### PR DESCRIPTION
**Pull request by JRosenkranz**
Added a llama3 model and speculator variant

### CODE REVIEW

File: fms_extras/models/paged_llama.py [hunk 1]
```diff
@@ -726,6 +728,22 @@ def factory(**kwargs):
...
+    norm_eps=1e-5,
+    nheads=32,
+    kvheads=8,
+    nlayers=32,
+    hidden_grow_factor=3.5,
+    multiple_of=1024,
+    max_expected_seq_len=8192,
+)
+
+models.register_model("paged_llama3", "8b", _llama_factory_factory((_8b_llama3_config)))
```
Comment (nairbv) [inline fms_extras/models/paged_llama.py]: I suspect `paged_llama3` isn't actually a different architecture vs `paged_llama`?

Comment (JRosenkranz) [reply fms_extras/models/paged_llama.py]: yes, its the same architecture, just was trying to differentiate. I am ok with keeping it as `paged_llama` as well.

Comment (nairbv) [reply fms_extras/models/paged_llama.py]: IMO we should probably keep the same arch name when the architecture doesn't change

Comment (JRosenkranz) [reply fms_extras/models/paged_llama.py]: Fixed in latest commit

File: fms_extras/models/paged_llama.py [hunk 2]
```diff
@@ -742,7 +742,7 @@ def factory(**kwargs):
     max_expected_seq_len=8192,
 )
 
-models.register_model("paged_llama3", "8b", _llama_factory_factory((_8b_llama3_config)))
+models.register_model("paged_llama", "8b", _llama_factory_factory((_8b_llama3_config)))
```
Comment (nairbv) [inline fms_extras/models/paged_llama.py]: better to re-use `_architecture_name` since we already have a constant for this

Comment (JRosenkranz) [reply fms_extras/models/paged_llama.py]: Fixed in latest commit

**Review by nairbv — APPROVED** [review]
might want to consider more granular variant name since '8b' is for "llama 3" (but leaving the architecture name the same).

**Comment by JRosenkranz** [comment]
> might want to consider more granular variant name since '8b' is for "llama 3" (but leaving the architecture name the same).

Fixed this to llama3.8b, but we may want to consider doing something like this for the rest of the variants in the future. Currently we will assume without llama3 is default llama2

**Pull request merged**